src/main/java/org/folio/clients/ConfigurationsClient.java

```diff
@@ -0,0 +1,74 @@
+package org.folio.clients;
+
+import io.vertx.core.json.DecodeException;
+import io.vertx.core.json.Json;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import org.apache.commons.collections4.CollectionUtils;
+import org.folio.service.mapping.processor.rule.Rule;
+import org.folio.util.OkapiConnectionParams;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import java.lang.invoke.MethodHandles;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static java.util.Collections.emptyList;
+import static org.folio.util.ExternalPathResolver.CONFIGURATIONS;
+import static org.folio.util.ExternalPathResolver.resourcesPathWithPrefix;
+
+@Component
+public class ConfigurationsClient {
+  private static final Logger LOGGER = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+  private static final String QUERY_EXPORT_PATTERN = "?query=code=DATA_EXPORT";
```

Comment:
It would be more handy if we can check for code and enabled= true. So that if they temporarily need to disable it there is a flexibility

Reply:
updated